src/Adapter/Driver/Mysqli/Connection.php

```diff
@@ -114,7 +114,6 @@ public function connect()
         $cipher = (isset($p['cipher'])) ? $p['cipher'] : '';
 
         $this->resource = $this->createResource();
-        $this->resource->init();
```

Comment:
This smells a bit 🤔

Reply:
https://wiki.php.net/rfc/deprecations_php_8_1#mysqliinit